app/models/manageiq/providers/openstack/network_manager/cloud_subnet.rb

```diff
@@ -1,4 +1,20 @@
 class ManageIQ::Providers::Openstack::NetworkManager::CloudSubnet < ::CloudSubnet
+  supports :create
+  supports :delete do
+    if ext_management_system.nil?
+      unsupported_reason_add(:delete_subnet, _("The subnet is not connected to an active %{table}") % {
```

Comment:
This unsupported reason should be called `delete`. Unfortunately this mistake exists in the original PR that you are backporting, but it would be good to fix it.